src/nodes.ts

```diff
@@ -22,31 +23,33 @@ export type PropertyKey = Identifier | Literal;
 export type PropertyValue = AssignmentPattern | BindingIdentifier | BindingPattern | FunctionExpression;
 export type StatementListItem = Declaration | Statement;
 
+type Nullable<T> = T | null;
```

Comment:
That was my original implementation. There were however 45 occurrences of `| null` and hence I looked for a sensible shorthand for that.